### DISCUSSION TITLE
A feature to turn on/off sub or dub anime

### DISCUSSION THREAD

**Discussion by Roswaal12** [aniyomiorg/aniyomi#709] [Ideas]
Can you add a feature where we can disable subbed or dubbed anime from viewing in extension? Something similar to source language option we have in extension settings.
![Screenshot_2022-08-06-16-13-00-32_423fe07c33ed17f15a4e92e5e3a15972](https://user-images.githubusercontent.com/86913887/183244581-15b463e7-7c94-4a92-97af-a5c49daa2668.jpg)

**Comment by Abinsounderraj** [comment]
Will add it on next update :-)

**Reply by Roswaal12** [reply]
It would be very helpful
